src/components/ImageCircle/ImageCircle.tsx

```diff
@@ -22,17 +26,20 @@ export default class ImageCircle extends React.Component<IProps> {
 	};
 
 	render () {
-		const ContainerTag: any = this.props.tag;
-
 		return (
-			<ContainerTag
+			<ClippedContent
```

Comment:
Now using `ClippedContent` internally so its consistent sizing and shape with Avatar initials.